Classes/Service/BigQueryService.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace t3n\FlowLog\Service;
+
+use Google\Cloud\BigQuery\BigQueryClient;
+use Google\Cloud\BigQuery\Dataset;
+use Google\Cloud\BigQuery\Table;
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Exception;
+
+/**
+ * @Flow\Scope("singleton")
+ */
+class BigQueryService
+{
+    /**
+     * @Flow\InjectConfiguration(package="t3n.FlowLog", path="bigQuery")
+     *
+     * @var mixed[]
+     */
+    protected $settings;
+
+    /**
+     * @var BigQueryClient
+     */
+    protected $bigQueryClient;
+
+    public function getClient(): BigQueryClient
+    {
+        if ($this->settings['keyFilePath'] === null || $this->settings['dataset'] === null || $this->settings['table'] === null) {
```

Comment:
Checks should check for empty values as well

Reply:
Good point - fixed!